releases/patch-releases.md

```diff
@@ -54,6 +54,14 @@ releases may also occur in between these.
 
 ## Detailed Release History for Active Branches
 
+### 1.21
+
+End of Life for **1.21** is **2022-04-30**
```

Comment:
I'm not sure is this the correct date, but I assumed one year from now, so tentatively it should be by end of April 2022.

Reply:
You're right.

Generally we may omit the actual day from the EOL date. I think this way we can simply state that the last patch release  is the one of April 2022. :thinking:

Reply:
@saschagrunert @hasheddan I thought about that as well, but I was not sure what to put in `schedule.yaml`. That file is supposed to be machine-readable, and so far we've only put dates in it. I thought about omitting day (e.g. just `2022-04`) or putting some string description, but I'm worried it could break users that depend on that file (if there are any).

I wanted to keep the consistency between `patch-releases.md` and `schedule.yaml`, so I decided to put some tentative date as we did before. If you have any idea how to workaround this, please let me know.

Reply:
Let's keep them for now and revisit this discussion later on.